app/controllers/support/providers_controller.rb

```diff
@@ -15,16 +15,19 @@ def create
       if @provider.save
         redirect_to support_provider_path(provider), flash: { success: "Provider was successfully created" }
       else
-        # The below code is to fix a mismatch of error messages
-        # for invalid forms in the support console.
-        @provider.errors.messages.each { |k, v|
-          case k
-          when :"sites.urn", :email, :telephone
-            @provider.errors.messages[k] = v.first.gsub("^", "")
-          else
-            @provider.errors.messages[k] = "#{k.to_s.gsub(/.\.^?/, ' ').humanize} #{v.first}"
-          end
-        }
+        # TODO: Move this into a form object, the error linking is currently broken and the code below violates
```

Comment:
![image](https://user-images.githubusercontent.com/470137/172857018-6ce2b9bb-f0eb-4c62-a7a1-07094938a62d.png)
Is it this?

Reply:
Yup carded up here https://trello.com/c/7ovcLHFw/143-publish-refactor-provider-form-in-support-console. The error linking wasn't really working properly to begin with.